apps/ledger-live-desktop/src/renderer/screens/account/AccountStakeBanner.tsx

```diff
@@ -19,7 +20,9 @@ export const AccountStakeBanner: React.FC<{
         return <EthereumStakeBanner account={account} />;
       case "cosmos":
       case "osmo":
-        return <CosmosStakeBanner account={account as CosmosAccount} />;
+        return (
+          <CosmosStakeBanner account={account as CosmosAccount} parenAccount={parentAccount} />
```

Comment:
typo: `parentAccount={parentAccount}`

Reply:
? The typo is already defined. Not sure to understand the comment.